src/app/components/RadioButton/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+
+export type Props = {
+  title: string;
+  description: string;
+  icon: IconType;
+  onClick: () => void;
+};
+
+interface IconTypeProps {
+  className: string;
+}
+
+type IconType = (props: IconTypeProps) => JSX.Element;
+
+export default function RadioButton({
```

Comment:
This doesn't feel like a RadioButton to me, it feels more like a card (I know we already have a card component, which should probably be renamed/moved because it is not for general use)

Reply:
You are right, especially without the "radio" behavior that makes no sense anymore. I chose to rename to "CardButton". The Card should be renamed yes, seems like a very specific use case.